pkg/codegen/templates/stdhttp/std-http-handler.tmpl

```diff
@@ -3,21 +3,27 @@ func Handler(si ServerInterface) http.Handler {
   return HandlerWithOptions(si, StdHTTPServerOptions{})
 }
 
+// ServeMux is an abstraction of http.ServeMux.
+type ServeMux interface {
```

Comment:
Does this need to be exported? Probably no big deal it is, but thought I'd ask

Reply:
It’s good since you use it in the method’s arguments.